tinygrad/runtime/graph/hsa.py

```diff
@@ -145,6 +151,7 @@ def __call__(self, input_rawbuffers: List[Buffer], var_vals: Dict[Variable, int]
       hsa.hsa_signal_wait_scacquire(self.finish_signal, hsa.HSA_SIGNAL_CONDITION_LT, 1, (1 << 64) - 1, hsa.HSA_WAIT_STATE_ACTIVE)
       et = time.perf_counter() - st
 
+    for prof in self.profile_info: Profiler.track(*prof)
```

Comment:
This looks slow, maybe that's the python overhead? Why isn't it going right to self.tracked_signals?